modules/sdk/gradle-plugins-defaults/CHANGELOG.markdown

```diff
@@ -1053,10 +1053,21 @@ files.
 
 ## 2.4.33 - 2017-02-14
 
+### Changed
+- [LPS-70494]: Update the [Liferay Gradle Plugins] dependency to version 3.2.4.
+
 ### Fixed
 - [LPS-67863]: Exclude test projects from `gradlew baseline -PsyncRelease`
 process.
-- [LPS-70494]: Update the [Liferay Gradle Plugins] dependency to version 3.2.4.
+
+## 2.4.34 - 2017-02-15
+
+### Changed
+- [LPS-70677]: Update the [Liferay Gradle Plugins] dependency to version 3.2.5.
+
+### Removed
+- [LPS-70677]: Remove exclusion of `com.liferay.portal` transitive dependencies
+from the `com.liferay.jasper.jspc` default dependency.
```

Comment:
@Ithildir Is the `com.liferay.jasper.jspc` dependency here in the `jspC` configuration? We specify it being in the `jspC` config for `gradle-plugins` and `gradle-plugins-jasper-jspc`, but not here. Just wanted to double check. Thanks!